openbas-model/src/main/java/io/openbas/database/model/Team.java

```diff
@@ -144,13 +159,15 @@ public List<Inject> getExercisesInjects() {
   }
 
   @JsonProperty("team_exercise_injects_number")
+  @Schema(description = "Number of injects of all simulations of the team")
   public long getExercisesInjectsNumber() {
     return getExercisesInjects().size();
   }
 
   @ArraySchema(schema = @Schema(type = "string"))
   @JsonProperty("team_scenario_injects")
   @JsonSerialize(using = MultiIdListDeserializer.class)
+  @Schema(description = "List of injects from all scenarios of the team")
```

Comment:
With a multiIdListDeserializer, this is only the list of ID's on client side, no ?

Reply:
Yes, fixed it !